### PR TITLE
About us updated

### LINKED ISSUE #3308: Update Planet information in Telescope About Page
**What would you like to be added**:
Create new About page with new content of what we would like to say about Telescope. On the Telescope website - we want to update the Planet information. 

**Why would you like this to be added**:
We need to update the about page.

### PR DESCRIPTION
**Pull request by sirinoks**
## Issue This PR Addresses

Fixes #3308 

## Type of Change
- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

Updated About us to explain what Telescope does, rather than focus in on what it copies. It's only a text change, no code, so things shouldn't break.

## Steps to test the PR

- `npm install`
- `cp config/env.staging .env`
- `npm run dev`
- Click on "About us" and see the changes!

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### Comments
Sorry this took so long, I had it ready on weekends but I was stuck on making a PR..

### CODE REVIEW

File: src/web/app/src/components/AboutPage.tsx [hunk 1]
```diff
@@ -107,7 +107,17 @@ const AboutPage = () => {
         you want to see it..
       </p>
 
-      <h2>What is a Planet?</h2>
+      <h2>What does Telescope do?</h2>
+      <p>
+        In essence, Telescope takes RSS blog post feeds, rebuilds those blog posts into HTML and
```
Comment (humphd) [inline src/web/app/src/components/AboutPage.tsx:112]: Can you add a like for `RSS` to https://en.wikipedia.org/wiki/RSS and maybe https://rss.com/blog/rss-feed-for-blog/ for "blog post feed"?

File: src/web/app/src/components/AboutPage.tsx [hunk 2]
```diff
@@ -107,7 +107,17 @@ const AboutPage = () => {
...
+        collects them in a single web page to display. It is able to handle various formatting, such
+        as code blocks or embedded videos. Telescope also collects data about those posts in the{' '}
+        <a href="https://api.telescope.cdot.systems/v1/status/">dashboard</a>. Blog posts on
+        Telescope come from contributors to Telescope. Therefore, you get to follow the
+        website&quot;s development on the website itself!
+      </p>
+
+      <h2>It all starts with a Planet</h2>
```
Comment (humphd) [inline src/web/app/src/components/AboutPage.tsx:120]: What about "started" vs. "starts", since we don't use the planet anymore, but it inspired the work.

**Review by humphd — CHANGES_REQUESTED** [review]
I like it.  A few comments.

File: src/web/app/src/components/AboutPage.tsx (hunk 1 above)
Comment (sirinoks) [reply src/web/app/src/components/AboutPage.tsx]: Yes.

File: src/web/app/src/components/AboutPage.tsx (hunk 2 above)
Comment (sirinoks) [reply src/web/app/src/components/AboutPage.tsx]: Changed.

**Review by dbelokon — DISMISSED** [review]

**Review by menghif — APPROVED** [review]
Looks good!

**Review by joelazwar — APPROVED** [review]

**Comment by tcvan0707** [comment]
@sirinoks can you please rebase it?

**Comment by humphd** [comment]
This needs to be squashed + rebased

**Review by menghif — APPROVED** [review]

**Review by AmasiaNalbandian — APPROVED** [review]

**Review by menghif — CHANGES_REQUESTED** [review]
You can't merge master into the branch.

**Review by tcvan0707 — APPROVED** [review]

**Review by JerryHue — APPROVED** [review]

**Review by tcvan0707 — DISMISSED** [review]

**Review by menghif — APPROVED** [review]

**Review by JerryHue — APPROVED** [review]

**Pull request merged**